src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -284,7 +284,7 @@ public static JsonDocument Parse(string json, JsonDocumentOptions options = defa
         ///   </para>
         /// </remarks>
         /// <exception cref="ArgumentException">
-        ///   <paramref name="reader"/> is using unsupported options.
+        ///   <paramref name="reader"/> contains unsupported options.
```

Comment:
Is this where we landed on the doc sentence?

From reading https://github.com/dotnet/runtime/pull/43601#discussion_r512140159, I thought we'd keep "is using unsupported options" across the board instead (and update docs accordingly).

Reply:
cc @tdykstra

Reply:
"is using" makes more sense to me.

Reply:
Since the current doc is "is using*" I assumed that is more correct since it was "contains*" originally, meaning it was already reviewed by the doc team (?)

Reply:
Since the docs have been reviewed after the original comments in code were ported, I assume docs win.

However, it looks a little messy:
- The [original code ](https://github.com/dotnet/corefx/commit/81e9045dca63ccea9b4fe31f8ac3a035b1b1f86d#diff-381cac507f4b825589eda03c662e02d9d41befdc6d307d6ec67e4a1c6deea963R40) used "contains" and is still using that
- When [originally ported](https://github.com/dotnet/dotnet-api-docs/pull/2471/files#diff-a22c4cb000916f16c8a84d78eb0e2888769b6eff19df0d6505edde5b85103a37R91) to dotnet-api-docs is still used "contains"
- Another [pass ](https://github.com/carlossanlop/dotnet-api-docs/commit/aecf3f1171911acdf790d7c97bfe18c78f0d956e#diff-a22c4cb000916f16c8a84d78eb0e2888769b6eff19df0d6505edde5b85103a37R305) was made to dotnet-api-doc changing from "is using" to "contains" although history doesn't show the "is using" change...

Reply:
I reverted the "contains" changes.

However, there are also similar "contains unsupported options" elsewhere for [JsonDocumentOptions](https://github.com/dotnet/runtime/blob/4f1412e1fdefc37944e9ea15170831fc01e4bc45/src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs#L45); I don't have a preference here.

Reply:
Is there an issue tracking the docs update, in dotnet-api-docs, to make the wording consistent (`is using unsupported options`)?

It's nice to have the XML comments in source match what's in the official docs/IntelliSense, but not absolutely necessary. The priority is getting the donet-api-docs to be accurate and consistent because **that** is the source of truth.